core/controllers/resources.py

```diff
@@ -95,7 +95,8 @@ def get(self, page_context, page_identifier, asset_type, encoded_filename):
                     page_context == feconf.ENTITY_TYPE_EXPLORATION or
                     page_context == feconf.ENTITY_TYPE_SKILL or
                     page_context == feconf.ENTITY_TYPE_TOPIC or
-                    page_context == feconf.ENTITY_TYPE_STORY):
+                    page_context == feconf.ENTITY_TYPE_STORY or
+                    page_context == feconf.ENTITY_TYPE_QUESTION):
```

Comment:
Should we have list of these in `feconf.py` and then do just `page_context in xyz`?

Reply:
This list is not going to increase later on, and is more or less a fixed list. Also, there are other entity types like 'Subtopics', which is checked separately above (and is technically a sub-entity under topics), and 'Voiceover Application', which is not related to images, also in feconf.
So, I'm not sure if we can add all of these into a common list. What do you think?

Reply:
Even if we don't want to add this to a common list in feconf.py, I think its a good idea to make a temporary list just in this function and do page_context in xyz for readability purposes

Reply:
Exactly, if we want to explicitly throw an error when the context is different you can add all these into one list (including subtopic), have `if page_context in context_list` and the inside this block have a separate `if` for checking the subtopic.

Reply:
Ok, done! Created `_SUPPORTED_PAGE_CONTEXTS` locally and used that to raise 404 if invalid.

Reply:
Thanks!